src/api/app/views/webui2/webui/package/_tabs.html.haml

```diff
@@ -0,0 +1,15 @@
+.bg-light
+  %ul.nav.nav-tabs.pt-2.px-3
+    - if @package.name == "patchinfo"
+      %li.nav-item= link_to('Details', patchinfo_show_path, id: 'tab-details', class: "nav-link#{' active' if current_page?(patchinfo_show_path)}")
```

Comment:
I am aware that the `if current_page? {...}` part is ugly. I am thinking if I should put this in a helper.

Reply:
hm the only thing thaz I find ugly about it is that the whitespace is in the interpolation...

And my personal preference is
```
link_to...do
  Link Text
end
```

Reply:
Also Im not sure what you gain by putting it on the same line as li, clarity it is not 😋